soperator/installations/example/variables.tf

```diff
@@ -274,6 +274,21 @@ variable "slurm_operator_version" {
   nullable    = false
 }
 
+# region PartitionConfiguration
+
+variable "partitionConfigType" {
+  description = "Could be default of custom"
+  nullable    = true
+  default     = "default"
+  type = string
+}
+
+variable "partitionRawConfig" {
+  description = "Configuration as list strings started with PartitionName"
+  type        = list(string)
+  nullable    = true
```

Comment:
1. The same as for `nullable = true` in `partitionConfigType`
2. It should be defaulted to `[]`, I suppose
3. It should have check for empty array in case of `custom` config type